qfieldsync/gui/cloud_transfer_dialog.py

```diff
@@ -538,13 +541,54 @@ def build_files_tree(self):
                 # the length of the stack and the parts is equal for file entries
                 if len(stack) == len(parts):
                     item.setData(0, Qt.UserRole, project_file)
-                    self.add_file_checkbox_buttons(item, project_file)
+                    is_offline_layer = (
+                        project_file.local_path_exists
+                        and str(project_file.local_path) in offline_layers_paths
+                    )
+                    self.add_file_checkbox_buttons(item, project_file, is_offline_layer)
                 else:
                     # TODO make a fancy button that marks all the child items as checked or not
                     pass
         self.filesTree.expandAll()
         # NOTE END algorithmic part
 
+    def _get_offline_layers(self):
+        offline_layers_paths = []
```

Comment:
```suggestion
    def _get_offline_layers_paths(self) -> List[str]:
    """
    Get the paths of project layers configured for offline editing.
    """
        offline_layers_paths = []
```

Maybe a little docstring could be nice here. Feel free to reword / enhance / densify.

Reply:
@gounux , done.